management_api_app/models/schemas/workspace_service.py

```diff
@@ -0,0 +1,59 @@
+from pydantic import BaseModel, Field
+
+from models.domain.workspace import Workspace
+
+
+def get_sample_workspace(workspace_id: str, spec_workspace_id: str = "0001") -> dict:
+    return {
+        "id": workspace_id,
+        "workspaceId": "7289ru33-7265-4b5f-9eae-a1a62928772e",
+        "resourceTemplateName": "guacamole",
+        "resourceTemplateVersion": "0.1.0",
+        "resourceTemplateParameters": {
+            "displayName": "my workspace service",
+            "description": "some description",
+         },
+        "deployment": {
+            "status": "not_deployed",
+            "message": "This resource is not yet deployed"
+        },
+        "deleted": False,
+        "resourceType": "workspace-service",
```

Comment:
same here -- let's use the enum (and the same applies for status above) -- less places to change when we change the "magic string"